### DISCUSSION TITLE
async_hooks dependency not found

### DISCUSSION THREAD

**Discussion by miainchambers** [prisma/prisma#3085] [Q&A] [answered]
I set up a new project to use Prisma with Vue but when I import PrismaClient into my component to be used it gives me a never ending error of async_hooks dependency can't be found in './node_modules/@prisma/client/runtime/index.js'.

I've ran npm install on async_hooks on my end but still getting the error and I have no idea how to fix it.

Any help it greatly appreciated.
![Screenshot 2020-07-20 at 17 45 27](https://user-images.githubusercontent.com/16454408/87963594-dc760b80-cab0-11ea-9559-f1f32b6ceaa6.png)

![Screenshot 2020-07-20 at 17 45 39](https://user-images.githubusercontent.com/16454408/87963597-dd0ea200-cab0-11ea-949d-e6fb9c5078bf.png)

**Comment by ryands17** [comment, marked as answer]
Hey @miainchambers 👋 
Unfortunately Prisma doesn't work on the frontend. If you're using just Vue, you would need to create a backend for Prisma to work with. If you're using Nuxt (server side rendering), you will be able to use Prisma on the Node side of things.

**Reply by miainchambers** [reply]
Yeah I realised last night! I started working on a backend API gateway and it started to work. Thanks anyway :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
